ReleaseProcess.md

```diff
@@ -0,0 +1,9 @@
+**Process for releasing a new version of VSCode extension - Deploy to Azure**
+
+1. Update the new version in package.json and package-lock.json 
+2. Add one liner for the new version in ChangeLog.md and make changes in README.(if required).
+3. Create a release branch with the version name. Example, for version v1.2.3, create a new branch releases/v1.2.3.
```

Comment:
create a new release branch only for major version. For example, there should be new branch for v1, v2 etc